streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -25,18 +24,19 @@
 public class ExecutionJobVertex {
 
   /**
-   * Unique id of operator(use {@link JobVertex}'s id). Used as jobVertex's id.
+   * Unique id of operator(use {@link JobVertex}'s id). Used as executionJobVertex's id.
    */
-  private final int jobVertexId;
+  private final int executionJobVertexId;
 
   /**
-   * Unique name of operator(use {@link StreamOperator}'s name). Used as jobVertex's name.
+   * Unique name of operator(use {@link StreamOperator}'s name). Used as executionJobVertex's name.
```

Comment:
What does `Used as executionJobVertex's name.` mean?

Reply:
refine this comment